src/nomad_catalysis/schema_packages/catalysis.py

```diff
@@ -0,0 +1,378 @@
+        categories=[CatalysisElnCategory],
+    )
+
+    preparation_details = SubSection(
+        section_def=Preparation, a_eln=ELNAnnotation(label='Preparation Details')
+    )
+
+    surface = SubSection(
+        section_def=SurfaceArea, a_eln=ELNAnnotation(label='Surface Area')
+    )
+
+    storing_institution = Quantity(
+        type=str,
+        shape=[],
+        description="""
+        The institution at which the sample is stored.
+        """,
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'Fritz-Haber-Institut Berlin / Abteilung AC',
+                    'Fritz-Haber-Institut Berlin / ISC',
+                    'TU Berlin / BasCat',
+                ]
+            ),
+        ),
+    )
+
+    catalyst_type = Quantity(
+        type=str,
+        shape=['*'],
+        description="""
+          A classification of the catalyst type.
+          """,
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'bulk catalyst',
+                    'supported catalyst',
+                    'single crystal',
+                    'metal',
+                    'oxide',
+                    '2D catalyst',
+                    'other',
+                    'unkown',
+                ]
+            ),
+        ),
+        links=['https://w3id.org/nfdi4cat/voc4cat_0007014'],
+    )
+
+    form = Quantity(
+        type=str,
+        shape=[],
+        description="""
+          classification of physical form of catalyst
+          """,
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(suggestions=['sieve fraction', 'powder', 'thin film']),
+        ),
+        links=['https://w3id.org/nfdi4cat/voc4cat_0000016'],
+    )
+
+    def populate_results(
+            self, archive: 'EntryArchive', logger) -> None:
+        '''
+        This function copies the catalyst sample information specified in the dict
+         quantities_results_mapping into the results archive of the entry.
+        '''
+
+        sample_obj = self
+
+        add_catalyst(archive)
+        quantities_results_mapping = {
+            'name': 'catalyst_name',
+            'catalyst_type': 'catalyst_type',
+            'preparation_details.preparation_method': 'preparation_method',
+            'surface.surface_area': 'surface_area',
+            'surface.method_surface_area_determination': 'characterization_methods',
+        }
+
+        for ref_attr, catalyst_attr in quantities_results_mapping.items():
+            value = get_nested_attr(sample_obj, ref_attr)
+            if value is not None:
+                try:
+                    setattr(
+                        archive.results.properties.catalytic.catalyst,
+                        catalyst_attr,
+                        value
+                    )
+                except ValueError:
```

Comment:
If this is still needed then maybe add an inline comment explaining why this workaround.

Reply:
removed and put general exception  error logger warning